### ISSUE TITLE
Error notification when trying to sign up with duplicate email

### ISSUE THREAD

**Issue by sungbae** [codefordenver/members#121]
This is a continuation of #107. Just need to provide some kind of error notification or some way to tell the user that duplicate emails are not allowed when signing up.

**Comment by dchao19** [comment]
It might be helpful to use Auth0 rules to link accounts across different social connectors (google vs. database vs. linkedin etc.) with this https://auth0.com/docs/rules/current, but this requires a paid subscription to Auth0.

Auth0 already prohibits user duplication by email within a connector (no two linkedin accounts with the same email will be able to join), but not between connectors.

I suppose it is also possible to do this linking on the database level - but I don't think it should throw an error. It should just merge the accounts.

**Comment by dchao19** [comment]
Willy and I investigated further into this, and decided it wasn't worth the implementation time and potential authentication overhead to either link accounts or prohibit duplicate accounts. 

We looked at using the email as the primary key for user entries in the database, and collecting all auth0 userIds that link to that email, but this introduces some significant overhead. We would either have to reach out to the Auth0 authentication api for user profile information (read: email) and then search the database, or we would have to iterate each user to match an auth0userId to one in a list of other ids.

We also looked at using auth0 rules to stop duplicate account matching, but the development debt from implementing this would be great. It would require another script to manage the deployment of the rule. 

Frankly, we decided this isn't too much of a problem. Duplicate accounts doesn't really cause any major database problems or any ux problems for the user. They can just sign out and sign in with the correct account.
